src/components/InputTime/AsString.tsx

```diff
@@ -50,6 +56,10 @@ const AsString: React.FC<AsStringProps> = ({
   >;
 
   useImperativeHandle(forwardedRef, () => localRef.current, []);
+  const [validity, setValidity] = useState(true);
+
+  const showDropdown: 'click' | undefined =
+    (customShowDropdown === 'click' && 'click') || undefined;
```

Comment:
nit: isn't this the same as `customShowDropdown === 'click' ? 'click' : undefined`? I think it might be confusing to have the `&& 'click'` right after checking that they have the same value

Reply:
I would agree with this. Seems like a tricky trick that would possibly confuse future developers (myself included!)

## 🧑‍⚖️

Reply:
Word.